Sources/NetworkProtection/PacketTunnelProvider.swift

```diff
@@ -994,9 +994,11 @@ open class PacketTunnelProvider: NEPacketTunnelProvider {
             connectionStatus = .connected(connectedDate: Date())
         }
 
-        guard !isKeyExpired else {
-            await rekey()
-            return
+        if !settings.disableRekeying {
```

Comment:
Please bear with me for a sec as I'm confused by this.  It certainly looks right to do this if there's a setting, but I'm lacking context about why this was added.

My question is two-fold and it's mostly to catch up with our plans:
1. Why was this added to begin with?
2. What's the plan for rekeying that makes us want to disable it optionally?

Reply:
This was a debug option setting that I added last year when I was talking with Aitor about the effects of removing rekeying - it's not used for prod users in any way. There's a discussion task in Asana hashing out the merits of removing rekeying, but I forget where it got to.

This change here is just a bugfix that tripped me up while testing this feature - I had the setting to disable rekeying turned on, which was causing the app to always get to this `isKeyExpired` check and never start the tunnel monitor.